hvplot/networkx.py

```diff
@@ -75,7 +75,7 @@ def _from_networkx(G, positions, nodes=None, cls=Graph, **kwargs):
     node_columns = defaultdict(list)
 
     # Unpack node positions
-    for idx, pos in sorted(positions.items()):
+    for idx, pos in positions.items():
```

Comment:
Wonder why I sorted here in the first place.

Reply:
Hard to tell why since there aren't any tests :(

Reply:
😞